src/main.c

```diff
@@ -1627,6 +1630,10 @@ int main(int argc, char *argv[], char *envp[])
 	enum qb_log_target_slot i;
 #endif
 
+#ifdef LIBPACEMAKER
+	crm_xml_init();
```

Comment:
We've deprecated crm_xml_init in pacemaker.  I'm not sure what the replacement for external users is at the moment, if any.  We'll have to think of something, though.  Looking into it.

Reply:
We're going to need to take care of https://projects.clusterlabs.org/T840 before this can be worked on.